wrappers/python/nntile/inference/llm_api_server.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from dataclasses import dataclass
+from typing import Annotated
+
+from fastapi import Body, FastAPI
+from pydantic import BaseModel, Field
+
+from nntile.inference.api_server_base import (
+    SimpleApiServerBase,
+    SimpleApiServerParams,
+)
+from nntile.inference.utils import dump_pydantic_model
+from nntile.model.generation.llm import GenerationMode, GenerationParams
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class SimpleLlmApiServerParams(SimpleApiServerParams):
+    pass
+
+
+class SimpleLlmApiServerGenerateRequest(BaseModel):
+    text: str
+    max_tokens: int = Field(
+        default=0,
+        description="How much tokens to generate including initial text",
+    )
+    mode: GenerationMode = Field(
+        default=GenerationMode.Greedy, description="Strategy for generation"
+    )
```

Comment:
Enum here is absolutely non-pythonic way to do things. Additional serialization efforts for `json.dump` is among consequences of this decision (`utils.py`).

Reply:
It is not very good. But logic is to use enums if you provide limited choice.
And actually you have validation on input and pretty schema view in swagger

Do you suggest just use strings all around? It will be less clear, won't it?

Note - it is not a production server, so it clearly not about performance. 
Just the thing you can show everybody as an example to use. And answer less questions like "How can I use it for inference?"

Reply:
No, it won't. Just define literal and `pydantic` will catch it.

Reply:
Agreed but I do not do such claims. Why do you think so?